src/main/java/hudson/plugins/throttleconcurrents/ThrottleJobProperty.java

```diff
@@ -57,6 +64,8 @@ public ThrottleJobProperty(Integer maxConcurrentPerNode,
                                List<String> categories,
                                boolean throttleEnabled,
                                String throttleOption,
+                               boolean limitOneJobWithMatchingParams,
```

Comment:
This change breaks the binary compatibility.
As  well as previous ones, so no complains